template.yml

```diff
@@ -119,6 +136,42 @@ objects:
   data:
     ca.crt: ${AWX_CA}
 
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: ${NAMESPACE}
+    name: ${NAME}-proxy-cookie
+  data:
+    session_secret: ${OAUTH_PROXY_SECRET}
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: ${NAME}-access
+  rules:
+    - apiGroups:
```

Comment:
Can you use the same indenting style than in the rest of the files?

```
rules:
- apiGroups:
  - ""
  resources:
...
```

Reply:
Done